install-python.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# This script installs python and related tooling
+#
+# This script must be run as root:
+# sudo sh install-python.sh
+
+if [[ $EUID -ne 0 ]]; then
+	echo "This script must be run as root" 1>&2
+	exit 1
+fi
+
+# Need to make python3 the default version
+
+sudo apt-get update
+
+sudo apt-get -y install python3-pip
+
+pip install --user pipenv
+pip intall virtualenv
```

Comment:
There's a typo here intall --> install. Also:
* this probably will need the `--user` switch as well
* we may just want to nix this line entirely and just let people use pipenv instead of manually dealing with virtualenvs